kirin/core/model.py

```diff
@@ -368,6 +368,17 @@ def find_stop(self, stop_id, order=None):
         return next((st for st in self.stop_time_updates
                      if st.stop_id == stop_id), None)
 
+    def find_first_departure(self):
```

Comment:
```suggestion
    def find_first_departure_date(self):
```
or "[...]_date`time`" depending on the minor comment below.